python/text/textindex.py

```diff
@@ -229,11 +250,27 @@ def extract_descriptions(input_filename, index, texts):
 
 
 # [START get_text]
-def get_text_from_file(vision, index, input_filename):
+def get_text_from_files(vision, index, input_filenames):
     """Call the Vision API on a file and index the results."""
-    with open(input_filename, 'rb') as image:
-        texts = vision.detect_text(image)
-        extract_descriptions(input_filename, index, texts)
+    texts = vision.detect_text(input_filenames)
+    for filename, text in texts.items():
+        extract_descriptions(filename, index, text)
+
+
+def batch(iterable, batch_size=4):
```

Comment:
Maybe we should define a 'constant' for the batch size, then set that constant as the default arg here rather than hardwiring the 4.  
Also, should we make the default batch size a bit larger, maybe 10?  (I tested with size 20 and it worked well).